### PR TITLE
Service plan module will replace app service plan AB#14277

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by pipalmic — APPROVED** [review]

File: service_plan/main.tf [hunk 1]
```diff
@@ -0,0 +1,20 @@
+terraform {
+  required_version = ">=1.0.9"
```
Comment (JeffreyVWT) [inline service_plan/main.tf]: Is this version correct?

File: service_plan/main.tf [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "=3.6.0"
```
Comment (JeffreyVWT) [inline service_plan/main.tf]: Why not use 3.48.0 ?

**Review by JeffreyVWT — CHANGES_REQUESTED** [review]

File: service_plan/main.tf (hunk 1 above)
Comment (ghost) [reply service_plan/main.tf]: @JeffreyVWT  nice spot could you check it out again

**Review by JeffreyVWT — APPROVED** [review]

**Comment by ghost** [comment]
[AB#14277](https://dev.azure.com/VolkerWesselsTelecom/ced71a30-926d-46d1-b3e0-f9afbc7b689f/_workitems/edit/14277)

**Pull request merged**